tests/common/mod.rs

```diff
@@ -220,6 +220,7 @@ impl PixiControl {
                 no_lockfile_update: false,
                 platform: Default::default(),
                 pypi: false,
+                sdist_resolution: Default::default(),
```

Comment:
Just a comment: one of the new linting rules of Bas' (in rattler) says to always use the specific default, e.g. `SDistResolution::default()` instead.

Reply:
Okay! Will change it :)